licenses.yaml

```diff
@@ -3093,7 +3093,7 @@ libraries:
 ---
 
 name: Apache Kafka
-version: 2.2.1
+version: 2.2.2
```

Comment:
Whichever PR gets merged between this and #9259, the notice section needs the year updated to 2019 to match the version notice file